### DISCUSSION TITLE
Plan to include a Regex based Replace option for StringBuilder

### DISCUSSION THREAD

**Discussion by egvijayanand** [dotnet/runtime#62731] [General] [answered]
Undoubtedly, StringBuilder is a performance booster when working with lot of strings.

But one thing that we as developers miss the most is Regex based Replace option that will make it even more efficient.

Kindly plan to include such an option in the upcoming release.

**Comment by HaloFour** [comment, marked as answer]
I'd suggest opening this discussion on the [runtime repository](https://github.com/dotnet/runtime/) as they manage the base class library and APIs.

**Reply by egvijayanand** [reply]
Thanks for the immediate response.

Will raise it in the .NET runtime repo.

**Reply by danmoseley** [reply]
@egvijayanand could you please open an issue in this repo? Use the api-proposal template. I'll close this discussion meantime.

**Reply by egvijayanand** [reply]
Please go ahead and close this discussion thread, have raised it as an issue in this runtime repo.

**Reply by danmoseley** [reply]
I realized I don't think discussions can be closed, but I marked an answer.

**Reply by CyrusNajmabadi** [reply]
It can't be closed.  But it can be locked (thus preventing people from trying to continue the discussion on the wrong issue/discussion vs the preferred one.  I've locked this one to ensure that that's the case here.

**Comment by CyrusNajmabadi** [comment]
Transferred to runtime team

**Comment by CyrusNajmabadi** [comment]
Now tracked at: https://github.com/dotnet/runtime/issues/62730
